pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_abstract_intervention.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.intervention import AbstractIntervention
+
+
+class TestAbstractIntervention(unittest.TestCase):
+    """Test the 'AbstractIntervention' class.
+    """
+
+    def test_construct(self):
+        start_time = 1
+        policy_duration = 10
+        case_threshold = 20
+        population = pe.Population()
```

Comment:
You could group these into a dict which you then unpack in all the function calls (like `AbstractIntervention(**intenvention_dict)` to prevent wordy repetition of code